src/main/java/com/redhat/patriot/network_simulator/example/image/DockerImage.java

```diff
@@ -0,0 +1,57 @@
+package com.redhat.patriot.network_simulator.example.image;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.core.command.BuildImageResultCallback;
+import com.redhat.patriot.network_simulator.example.files.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+public class DockerImage {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DockerImage.class);
+
+    private DockerClient dockerClient;
+
+    public DockerImage(DockerClient dockerClient) {
+        this.dockerClient = dockerClient;
+    }
+
+    public void buildImage(String tag, String path) {
+
+        FileUtils fileUtils = new FileUtils();
+
+        if (new File(DockerImage.class.getClassLoader().getResource(path).getFile()).exists()) {
```

Comment:
I'm not sure I understand this line, wouldn't is be better to save this file into some variable, you're opening it again later.